app/views/issues/_table.html.erb

```diff
@@ -1,5 +1,5 @@
-<% cache ['issues-table', @columns, issues.map(&:id), @issues.map(&:updated_at).map(&:to_i).sort.last] do %>
-  <table class="table table-striped tbl-issues js-tbl-issues">
+<% cache ['index-table', @columns, issues.map(&:id), @issues.map(&:updated_at).map(&:to_i).sort.last] do %>
```

Comment:
Not sure if we need to change cache keys. We are making them look more similar with this change. Different names is a good thing here.

Reply:
✅